deploy/terraform/terraform-units/modules/sap_system/common_infrastructure/storage_accounts.tf

```diff
@@ -31,9 +31,50 @@ resource "azurerm_storage_account" "sapmnt" {
   account_tier                    = "Premium"
   account_replication_type        = "ZRS"
   account_kind                    = "FileStorage"
-  enable_https_traffic_only       = false
+  enable_https_traffic_only       = true
```

Comment:
For Azure Files NFS this has to be false

https://docs.microsoft.com/en-us/azure/storage/files/storage-troubleshooting-files-nfs#cause-2-secure-transfer-required-is-enabled